<launch>
  <arg name="mav_name" default="firefly" />
  <arg name="robot_namespace" default="scout"/>
  <group ns="$(arg robot_namespace)">
    <node name="controller_node" pkg="my_mpc" type="controller_node" respawn="true" clear_params="true" output="screen">
      <rosparam file="$(find my_mpc)/config/gazebo_test.yaml" />
    </node>
  </group>

</launch>
